apps/front-end/pages/auth/sign-up/organization/index.tsx

```diff
@@ -0,0 +1,5 @@
+import React, { useState } from 'react';
+import OrganizationPage from '../../../../../../libs/ui/pages/auth/sign-up/organization';
```

Comment:
```suggestion
import OrganisationPage from '../../../../../../libs/ui/pages/auth/sign-up/organisation';
```

Organization -> Organisation